mu/logic.py

```diff
@@ -855,6 +856,71 @@ def connect_to_status_bar(self, status_bar):
             for mode in self.modes.values():
                 device_selector.device_changed.connect(mode.device_changed)
 
+    def detect_theme_preference(self):
+        """Detect OS dark/light theme preference.
+        Currently implemented for Mac, Windows, GNOME. For any other
+        operating systems, it will always return "day"
+        """
+        # Implementations inspired by the following Stackoverflow submissions:
+        # https://stackoverflow.com/questions/65294987/detect-os-dark-mode-in-python
+
+        def macos_prefers_dark_theme(self):
+            if sys.platform == "darwin":
+                cmd = "defaults read -g AppleInterfaceStyle"
+                result = subprocess.run(cmd.split())
+                return result.return_code == 0
```

Comment:
It will work fine without it, but since the function is meant to return a boolean maybe we want to close it with a `return False`? Instead of the implicit `return None`.